packages/networks/bin/doc.js

```diff
@@ -0,0 +1,24 @@
+const { networks } = require('../dist')
+
+const parseNetwork = ({ name, id, unlockAddress, explorer, nativeCurrency }) =>
+  `
+## ${name}
+
+- chainId: ${id}
+- native currency: ${nativeCurrency.name} (${nativeCurrency.symbol})
+- unlockAddress: \`[${unlockAddress}](${explorer.urls.address(unlockAddress)})\`
```

Comment:
There is also description added in https://github.com/unlock-protocol/unlock/pull/9183 

Let me merge it and then you can rebase?

Reply:
You can rebase now :D